vllm/v1/metrics/loggers.py

```diff
@@ -128,6 +128,12 @@ def __init__(self, vllm_config: VllmConfig):
             documentation="GPU KV-cache usage. 1 means 100 percent usage.",
             labelnames=labelnames).labels(*labelvalues)
 
+        self.gauge_total_tokens_in_queue = prometheus_client.Gauge(
+            name="vllm:total_tokens_in_queue",
```

Comment:
can we call this `num_tokens_waiting` for consistency with `num_requests_waiting`?